### PR TITLE
prepare for 4.1

### PR DESCRIPTION
**Pull request by nick-zh**

### CODE REVIEW

File: stubs/RdKafka/KafkaErrorException.php [hunk 1]
```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace RdKafka;
+
+class KafkaErrorException  extends \Exception
```
Comment (kwn) [inline stubs/RdKafka/KafkaErrorException.php:5]:
```suggestion
class KafkaErrorException extends \Exception
```

**Comment by kwn** [comment]
Thanks. Looking forward to converting draft into PR.

File: .travis.yml [hunk 2]
```diff
@@ -7,7 +7,7 @@ php:
 
 before_install:
   - sudo apt-get -y install gcc scala zookeeperd git
-  - git clone --depth 1 --branch v1.1.0 https://github.com/edenhill/librdkafka.git
+  - git clone --depth 1 --branch v1.4.0 https://github.com/edenhill/librdkafka.git
```
Comment (nick-zh) [inline .travis.yml:10]:
```suggestion
  - git clone --depth 1 --branch v1.5.2 https://github.com/edenhill/librdkafka.git
```

**Comment by nick-zh** [comment]
@kwn cannot merge myself :smile: need your approval :pray:

**Review by kwn — APPROVED** [review]

**Comment by nick-zh** [comment]
No worries, thx man!

**Pull request merged**